### PR TITLE
Add mkdir

### PR DESCRIPTION
**Pull request by yujinz**

### CODE REVIEW

File: _commands/basics/mkdir.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+--
+
+`mkdir` is used to create directories (if they do not already exist) on a file system.
+
+The general format is
+
+~~~ bash
+$ mkdir [options] directories
+~~~
+
```
Comment (mterwill) [inline _commands/basics/mkdir.md]: Please add a line after this one:

    

So ``/reference`` doesn't show your whole command.

**Comment by ppannuto** [comment]
Note: Replaces #10

**Comment by yujinz** [comment]
Format fixed. Is there any further modification needed to be done?

**Comment by ppannuto** [comment]
This looks good! Thanks :)

**Pull request merged**